### PR TITLE
Updating the Google Maps Extension

### PR DESCRIPTION
**Pull request by chrismessina**
### Changed

- Upgrade dependencies.
- Update extension icon.
- Renamed actions and refined UI text.
- Replaced emoji with Raycast icons.
- Added a Quick Search command.
- Added optional location search history.
- Added comments throughout.

## Screenshots

google-maps-search-1
google-maps-search-2
google-maps-search-3
google-maps-search-4
google-maps-search-5

## Checklist

- [X] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [X] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [X] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [X] I checked that files in the `assets` folder are used by the extension itself
- [X] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by ratoru** [comment]
Thank you for putting in the work to update this extension @chrismessina! The changes all look good to me.

File: extensions/google-maps-search/package.json [hunk 1]
```diff
@@ -2,91 +2,123 @@
...
+    "chrismessina"
+  ],
+  "keywords": [
+    "directions",
+    "navigation"
   ],
   "license": "MIT",
   "commands": [
     {
-      "name": "travelHome",
```
Comment (pernielsentikaer) [inline extensions/google-maps-search/package.json:17]: Can we keep the old `name`? It is the unique ID of the command where ranking, aliases, and hotkeys are saved. You can, of course, change the title 🙂

File: extensions/google-maps-search/package.json [hunk 2]
```diff
@@ -2,91 +2,123 @@
...
-      "title": "Travel To",
+      "name": "getMeSomewhere",
+      "title": "Get Me Somewhere",
       "subtitle": "Google Maps",
-      "description": "Open Google Maps with directions to a destination.",
+      "description": "Get directions to a destination.",
       "mode": "view"
     },
     {
-      "name": "find",
```
Comment (pernielsentikaer) [inline extensions/google-maps-search/package.json:31]: Can we keep the old `name`? It is the unique ID of the command where ranking, aliases, and hotkeys are saved. You can, of course, change the title 🙂

File: extensions/google-maps-search/package.json [hunk 3]
```diff
@@ -2,91 +2,123 @@
...
-      "name": "travelHome",
-      "title": "Travel Home",
+      "name": "getMeHome",
+      "title": "Get Me Home",
       "subtitle": "Google Maps",
       "description": "Open Google Maps with directions from your current location to your home.",
       "mode": "no-view"
     },
     {
-      "name": "travelTo",
```
Comment (pernielsentikaer) [inline extensions/google-maps-search/package.json:24]: Can we keep the old `name`? It is the unique ID of the command where ranking, aliases, and hotkeys are saved. You can, of course, change the title 🙂

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you 🙂

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/google-maps-search/package.json (hunk 1 above)
Comment (chrismessina) [reply extensions/google-maps-search/package.json:17]: Done!

File: extensions/google-maps-search/package.json (hunk 2 above)
Comment (chrismessina) [reply extensions/google-maps-search/package.json:31]: Done!

File: extensions/google-maps-search/package.json (hunk 3 above)
Comment (chrismessina) [reply extensions/google-maps-search/package.json:24]: Done!

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**